### PR TITLE
Added Block ads for MRAID

### PR DESCRIPTION
**Pull request by killer7Mod**

### CODE REVIEW

File: app/src/main/java/tw/fatminmin/xposed/minminguard/Main.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
             new Aotter(), new AppBrain(), new Applovin(), new Appnext(), new Avocarrot(), new Bonzai(), new Chartboost(), new Clickforce(), new Domob(), new Facebook(),
             new Freewheel(), new Flurry(), new GoogleGmsDoubleClick(), new Hodo(), new Inmobi(), new Intowow(), new Ironsource(), new KuAd(), new mAdserve(), new Madvertise(),
             new MasAd(), new MdotM(), new Millennial(), new Mobclix(), new MobFox(), new MoPub(), new Nend(), new Og(), new Onelouder(), new OpenX(), new SmartAdserver(),
-            new Startapp(), new Tapfortap(), new TWMads(), new UnityAds(), new Vpadn(), new Vpon(), new Vungle(), new Waystorm(), new Yandex(),
+            new Startapp(), new Tapfortap(), new TWMads(), new UnityAds(), new Vpadn(), new Vpon(), new Vungle(), new Waystorm(), new Yandex(),new Mraid(),new Mraid2(),
```
Comment (chiehmin) [inline app/src/main/java/tw/fatminmin/xposed/minminguard/Main.java]: make blockers lexical ordered

**Comment by MSkjel** [comment]
I am working on something cool that might negate the need for hard coding new ad blockers. It's not quite ready yet

**Comment by chiehmin** [comment]
> I am working on something cool that might negate the need for hard coding new ad blockers. It's not quite ready yet

Cool! I am looking forward to it!

**Comment by killer7Mod** [comment]
Ok, i will wait for new updates

**Comment by MSkjel** [comment]
Ok, that wont be ready for a while, so I'm going to accept this pull request. Next time, make sure you respect the formatting of the other Adblockers and the alphabetical order of the Adblocker list

**Pull request merged**